src/main/java/randoop/test/RegressionCaptureGenerator.java

```diff
@@ -253,4 +277,44 @@ public RegressionChecks generateTestChecks(ExecutableSequence sequence) {
     }
     return checks;
   }
+
+  /**
+   * Returns true if the method:
+   *
+   * <ul>
+   *   <li>has one formal parameter
```

Comment:
These implementation details can be converted to // comments in the method body.